Source/relay/TourGuide/Items of the Month/2020/Cargo Cultist Shorts.ash

```diff
@@ -5,25 +5,31 @@ void IOTMCargoCultistShortsGenerateResource(ChecklistEntry [int] resource_entrie
  
 	if (!get_property_boolean("_cargoPocketEmptied")) {
         string image_name = "__item cargo cultist shorts";
+
+        boolean [int] empty_pockets;
```

Comment:
oh man this has been annoying for so long. thanks. also, randomly, since this is not something that often gets mentioned -- if you don't have bean's cargo shorts UI you should get it! it's a nice UI. https://kolmafia.us/threads/cargo-shorts-gui.25416/

Reply:
I do not have it, thanks!